### PR TITLE
Add note to example about using with cuQuantum Appliance

### PR DESCRIPTION
**Pull request by eliottrosenberg**

### CODE REVIEW

File: docs/tutorials/gcp_gpu.md [hunk 1]
```diff
@@ -190,7 +190,8 @@ the code directly into the REPL, or paste the code in a file. See the documentat
 [here](https://quantumai.google/reference/python/qsimcirq/QSimOptions) for Options 1
 and 2 or [here](https://docs.nvidia.com/cuda/cuquantum/appliance/cirq.html) for Option 3.
 Make sure to change `xx` to 0 for Option 1, 1 for Option 2, or the number of GPUs for
-Option 3.
+Option 3. For option 3, note that `QSimOptions` has a `disable_gpu` flag instead of a
+`use_gpu` flag.
```
Comment (95-martin-orion) [inline docs/tutorials/gcp_gpu.md:194]: Where are you seeing this? I only see `use_gpu`:
https://github.com/quantumlib/qsim/blob/0f75bd98d6acecb43ef8d19f89cfe8ee6b57c486/qsimcirq/qsim_simulator.py#L61-L62

Comment (eliottrosenberg) [reply docs/tutorials/gcp_gpu.md:194]: @95-martin-orion https://docs.nvidia.com/cuda/cuquantum/latest/appliance/cirq.html

Comment (eliottrosenberg) [reply docs/tutorials/gcp_gpu.md:194]: It's in NVIDIA's modified version of qsim.

Comment (95-martin-orion) [reply docs/tutorials/gcp_gpu.md:194]: Ah. In that case, these have diverged enough that I recommend splitting the example:
```python
# Option 1 (mode=0) or Option 2 (mode=1)
gpu_options = qsimcirq.QSimOptions(use_gpu=True, gpu_mode = mode, max_fused_gate_size=4)
# Option 3 (number of GPUs = `num_gpus`)
gpu_options = qsimcirq.QSimOptions(disable_gpu=False, gpu_mode = num_gpus, max_fused_gate_size=4)
```

**Review by 95-martin-orion — CHANGES_REQUESTED** [review]

File: docs/tutorials/gcp_gpu.md (hunk 1 above)
Comment (eliottrosenberg) [reply docs/tutorials/gcp_gpu.md:194]: Done, thanks!

**Review by 95-martin-orion — APPROVED** [review]

**Pull request merged**